frontend/src/scenes/dashboard/dashboardLogic.ts

```diff
@@ -730,6 +753,9 @@ export const dashboardLogic = kea<dashboardLogicType>({
             }
         },
         loadDashboardItemsSuccess: () => {
+            // I didn't want any parameters but kea does :)
+            sharedListeners.reportLoadTiming(null, async () => {}, {} as { type: string; payload: any }, null)
```

Comment:
I think you can do something like this instead:
```suggestion
        loadDashboardItemsSuccess: (...args) => {
            sharedListeners.reportLoadTiming(...args)
```

Reply:
Ah, cool 🙌